### PR TITLE
Release/0.5.0

### PR DESCRIPTION
**Pull request by alexanderdean**

### CODE REVIEW

**Comment by chuwy** [comment]
@alexanderdean I think this PR is ready for review. Travis should be green as soon as schema-ddl will be published.

File: src/main/scala/com.snowplowanalytics/schemaguru/cli/DdlCommand.scala [hunk 1]
```diff
@@ -237,7 +241,8 @@ object DdlCommand extends GuruCommand {
    */
   private def getFileName(flatSelfElems: SelfDescInfo): (String, String) = {
     // Make the file name
-    val file = flatSelfElems.name.replaceAll("([^A-Z_])([A-Z])", "$1_$2").toLowerCase.concat("_1")
+    val version = "_".concat(flatSelfElems.version.replaceAll("-[0-9]-[0-9]", ""))
```
Comment (alexanderdean) [inline src/main/scala/com.snowplowanalytics/schemaguru/cli/DdlCommand.scala]: This regex is quite buggy:

```
scala> "-0-1".replaceAll("-[0-9]-[0-9]", "")
res0: String = ""

scala> "-0-10".replaceAll("-[0-9]-[0-9]", "")
res1: String = 0

scala> "-10-0".replaceAll("-[0-9]-[0-9]", "")
res2: String = -10-0
```

Can you fix and write some tests please?

**Comment by alexanderdean** [comment]
Can we get a blog post for this release please @chuwy ? It can be a short one (new features/bugs, plus how to upgrade).

**Comment by alexanderdean** [comment]
This PR is top priority Tues.

File: src/test/scala/CliSpec.scala [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import com.snowplowanalytics.schemaddl.SchemaData.SelfDescInfo
+
+class CliSpec extends Specification { def is = s2"""
+  Check CLI specification
+    generate DDL filename  $e1
+  """
+
+  def e1 = {
+    val fileName = ("com.acme", "event_name_1")
+    DdlCommand.getFileName(SelfDescInfo("com.acme", "event_name", "1-00-0")) must beEqualTo(fileName) and(
```
Comment (alexanderdean) [inline src/test/scala/CliSpec.scala]: Oh nice, I never knew about `and(` in Specs2

**Pull request merged**